platforms/core-runtime/logging/src/integTest/groovy/org/gradle/internal/logging/console/taskgrouping/AbstractConsoleDeprecationMessageGroupedTaskFunctionalTest.groovy

```diff
@@ -18,20 +18,14 @@ package org.gradle.internal.logging.console.taskgrouping
 
 import org.gradle.integtests.fixtures.console.AbstractConsoleGroupedTaskFunctionalTest
 
-import static org.gradle.util.internal.TextUtil.normaliseFileSeparators
-
 abstract class AbstractConsoleDeprecationMessageGroupedTaskFunctionalTest extends AbstractConsoleGroupedTaskFunctionalTest {
 
     private static final String JAVA_SRC_DIR_PATH = 'src/main/java'
 
-    def setup() {
-        executer.expectDeprecationWarning()
-    }
-
     def "compiler warnings emitted from compilation task are grouped"() {
         given:
         def javaSourceFile = file("$JAVA_SRC_DIR_PATH/MyClass.java")
-        def normalizedJavaSourceFilePath = normaliseFileSeparators(javaSourceFile.absolutePath)
+        def expectedOutput = "${javaSourceFile.absolutePath}:4: warning: [deprecation] Legacy in unnamed package has been deprecated"
```

Comment:
💭 I've dropped the file path normalization here, as it caused a test failure (the incoming path is not normalized), and also realizing that normalization doesn't make sense here: we are on the same platform, we should see a path native to the platform where the test runs.